Adds a Quick Reference section to Staking/Overview

hyd628 opened the pull request:
Initial commit. Adds the quick reference section to the Staking/Overview page that can be linked to provide a quick reference on the most commonly asked staking related questions. 

Also updates the max_nom_per_col global variable for Moonriver to match today's update.

eshaben reviewed (changes requested):
I was planning on getting rid of the staking-phase-4 message all together. But this is kind of nice that it updates it to a different message about the staking dashboard.

If we add links to the dashboard, we should probably add links to the tutorials or something so users know what to do when they get to the dashboard. Mabye? I don't know 🤷‍♀️ 

The staking-phase-4 message is in a few other places, it should be removed and replaced with the new post-launch message in all places. It exists in: staking/overview.md, stake.md, and stakingprecompile.md

hyd628 commented:
Thanks @eshaben , I have updated the PR to change the snippets in the other files, and rearranged the information a bit, and added links to our staking tutorials.

eshaben commented on snippets/text/moonriver-launch/post-launch.md:
```diff
@@ -0,0 +1 @@
+The Moonriver network has completed its launch process, and the staking feature is open to users that wish to participate. 
```
I liked it before because I thought it had relevant information like staking dapp instructions, and I see why you removed that part, but without it it seems kinda unnecessary

eshaben commented on staking/overview.md:
```diff
@@ -69,6 +85,6 @@ Where `amount_due` is the corresponding inflation being distributed in a specifi
+--8<-- 'text/moonriver-launch/post-launch.md'
 
---8<-- 'text/moonriver-launch/staking-phase-4.md'
+You can start interacting with staking functions through this [dashboard](https://apps.moonbeam.network/moonriver). To do so, you can check [this guide](/staking/stake/) or [this video tutorial](https://www.youtube.com/watch?v=maIfN2QkPpc).
```
I would link to this tutorial since it uses the staking dapp: https://moonbeam.network/tutorial/stake-movr/ instead of `/staking/stake` since it doesnt

eshaben commented on staking/overview.md:
```diff
@@ -15,6 +15,21 @@ The collators' set (nodes that produce blocks) are selected based on their stake
 
+## Quick Reference {: #quick-reference }
+
+* Minimum Nomination Amount: {{ networks.moonbase.staking.min_nom_stake }} MOVR/DEV
+* Max Eligible Nominators Per Collator: Top {{ networks.moonriver.staking.max_nom_per_col }} nominators by size of stake per collator are eligible for staking rewards (nominators not in the top {{ networks.moonriver.staking.max_nom_per_col }} do not receive any rewards)
+* Bonding Duration: {{ networks.moonriver.staking.bond_lock }} rounds ({{ networks.moonriver.staking.round_blocks }} blocks/round, time per round is approximately 1 hour)
+* Unbonding Duration: {{ networks.moonriver.staking.bond_lock }} rounds 
+* Reward Payout Time: Rewards are distributed automatically to the free balance {{ networks.moonriver.collator_timings.rewards_payouts.rounds }} rounds after they are earned
+* Aggregate Nominator Rewards: 50% of total inflation
+* Nominator Rewards: Variable; it's the aggregate nominator rewards distributed over all eligible nominators, taking into account the relative size of stakes; [More Information](/staking/overview/#reward-distribution)
+* Collator Commission: Fixed at 20% of total inflation, separate from the nominator reward pool
+* Slashing: No slashing currently (can be changed with governance); collator rewards are paid as part of produced blocks, so offline or poorly performing collators that don’t produce blocks will pay out less or no rewards
+* Collator Information: [Link1](https://moonriver.subscan.io/validator) [Link2](https://moonbeam-explorer.netlify.app/stats/miners/)
+* Managing Staking and Nomination Related Actions: [Link](https://apps.moonbeam.network/moonriver)
```
The formatting for this list needs to be updated. For an example of a formatted list, check out `snippets/text/staking/staking-definitions`

But in general the list should be formatted something like this:
- **Item one** - this is something
- **Item two** - this is something else

devrel-github-deploy replied:
Thanks. I've removed the snippets now and changed the list formatting. 

I've also slightly modified the parts which used to have the Moonriver launch snippet to make it more explicit the relevant staking information is applicable to both Moonriver and Moonbase Alpha to account for removing the Moonriver launch snippet.

hyd628 commented:
Thanks @eshaben . 

Updated the guide link, list formatting and removed the Moonriver launch snippets.

eshaben reviewed (changes requested):
added a few suggestions for more descriptive link text 🤷‍♀️ 

but other than that LGTM ✨

devrel-github-deploy commented:
Thanks @eshaben , have updated the link names.

hyd628 commented:
Thanks @eshaben, have updated the link names.

eshaben reviewed (approved):
Looks good! This will be super useful! 💪

albertov19 reviewed (changes requested):
Left some comments, overall it looks good.S

hyd628 commented:
@albertov19 Thanks. 

Added support for tabs to Quick Reference.

Set Moonriver as the default tab in all staking section pages.

Added variables for inflation related numbers.
